app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+  def index
```

Comment:
Following the idea of just making a single call with all the info we want (see my comment in your facade). Here is a way we could get all the info with a single call
```
def index
    results = FoodFacade.foods_by_keyword(params[:foods])
    @foods      = results[1]
    @total_hits = results[0]
  end
```